### PR TITLE
added pdf generate in python

### PR DESCRIPTION
**Pull request by aryabyte21**
## What does this PR do?

Generate PDF Template for python

## Test Plan

-

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: python/generate-pdf/Readme.md
Comment (loks0n) [inline python/generate-pdf/Readme.md:1]: Please rename this file to `README.md`

File: python/generate-pdf/src/main.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    print(f"Generated fake order: {fake_order}")
+
+    pdf_buffer = create_pdf(fake_order)
+    print("PDF created.")
+    
+    # If you want to write the PDF to a file:
+    # with open("output.pdf", "wb") as f:
+    #     f.write(pdf_buffer)
+    
+    return pdf_buffer
```
Comment (loks0n) [inline python/generate-pdf/src/main.py:12]: This won't work, we need to use res.send() with the requisite headers. Please see examples of the template in other runtimes

File: python/generate-pdf/Readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| Entrypoint        | `src/main.py`         |
+| Build Commands    | `pip install -r requirements.txt` |
+| Permissions       | `any`                 |
+| Timeout (Seconds) | 15                    |
+
+## 🔒 Environment Variables
+
+No environment variables required.
+
+> Note: Ensure you have a `requirements.txt` file in the project directory containing all the necessary packages (like `Faker`, `pdfkit`, etc.).
```
Comment (loks0n) [inline python/generate-pdf/Readme.md:34]: Since we provide this we don't need this line :)

**Review by loks0n — CHANGES_REQUESTED** [review]
Thanks for the contribution! Can you provide a video showing the function working.

Please note this contribution will not count towards hacktoberfest as it does not have a requisite issue.

File: python/generate-pdf/src/main.py (hunk 1 above)
Comment (aryabyte21) [reply python/generate-pdf/src/main.py:12]: ok got it!

**Comment by loks0n** [comment]
Closing as we don't have a relevant issue for this

**Pull request closed without merging**